packages/terra-layout/src/_LayoutSlidePanel.jsx

```diff
@@ -68,6 +68,7 @@ class LayoutSlidePanel extends React.Component {
   }
 
   componentDidMount() {
+    document.createElement('main'); // shim for <main> tag not being recognized in IE
```

Comment:
How does this work? Is it just magic?

Reply:
https://weblog.west-wind.com/posts/2015/Jan/12/main-HTML5-Tag-not-working-in-Internet-Explorer-91011

Reading through that link, I think we may be able to drop the JS part and just keep the CSS setting `main { display: block}` I think the element creation may only be needed for IE8 and below is not included in our browser support list.

Reply:
I'll check it out.

Reply:
![image](https://user-images.githubusercontent.com/13922343/49545801-48880980-f8a4-11e8-9fc5-242cb5ba1347.png)

This is what I get in IE11 if I remove the JS line.

Reply:
ah, good testing. Ok, let's leave the script.